hoss/projection_utilities.py

```diff
@@ -478,6 +486,45 @@ def get_x_y_extents_from_geographic_points(
 
     finite_x = points_x[np.isfinite(points_x)]
     finite_y = points_y[np.isfinite(points_y)]
```

Comment:
Are you sure, that x and y locations will always have the same missing values?
If some are infinite in x but not y, everything below won't work I don't think.

Reply:
You want to make a mask of where both x and y are finite.

``` python
finite_mask = np.isfinite(points_x) & np.isfinite(points_y)
finite_x = points_x[finite_mask]
finite_y = points_y[finite_mask]
```